resources/views/view.blade.php

```diff
@@ -0,0 +1,44 @@
+@extends('layout.master')  
+@section('Viewpost')
+<div> 
+    
+    <div>
+        <div>
+            <h2 class="mt-6 text-xl font-semibold text-gray-900 dark:text-white">My Posts</h2>
+
+            <form action="{{route('POSTS.return')}}" method="GET" >
+                @csrf
+                <button class="mt-4 text-gray-500 dark:text-gray-400 c leading-relaxed" > Create Posts </button>
+                </form>
+        </div>
+
+        <div  style= "padding:10px; margin:10px" > 
+            <h1 class="mt-6 text-xl font-semibold text-gray-900 dark:text-white" > All posts  </h1>
+
+            @foreach($posts as $posts)
```

Comment:
If you can please transfer the lists to where your create form is currently located.
Make it look like Facebook or Reddit or any other social media platform.